sdk/appconfiguration/azure-data-appconfiguration-perf/pom.xml

```diff
@@ -27,7 +27,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-data-appconfiguration</artifactId>
-      <version>1.2.0-beta.2</version> <!-- {x-version-update;com.azure:azure-data-appconfiguration;current} -->
+      <version>1.2.0-beta.1</version> <!-- {x-version-update;beta_com.azure:azure-data-appconfiguration;dependency} -->
```

Comment:
Why is this using beta dependency? Perf tests should be able to use `current` version.

Reply:
The performance test used the newest beta version to do the test. I might need to think about changing it to the GA version